### PR TITLE
Unfund known markets

### PR DESCRIPTION
**Pull request by kongzii**
## Summary by CodeRabbit

- **New Features**
	- Added new filtering options in market data retrieval based on opening time and liquidity.
	- Enhanced market withdrawal functionality with detailed logging.
	- Improved market replication process and introduced a feature to extend market closing time.
	- Introduced a new attribute `liquidityMeasure` to the `OmenMarket` class for better data representation.

- **Refactor**
	- Streamlined market unfunding process for better efficiency.

- **Chores**
	- Removed unused parameter from market data fetching function.

### CODE REVIEW

File: examples/replicate_markets/agent_example.py [hunk 1]
```diff
@@ -36,12 +37,19 @@ def run(
...
 
         logger.info(
-            f"Finalising, resolving anc claiming back xDai from existing markets replicated by {keys.bet_from_address}"
+            f"Finalising, resolving and claiming back xDai from existing markets replicated by {keys.bet_from_address}."
         )
         omen_finalize_and_resolve_and_claim_back_all_markets_based_on_others_tx(
             from_private_key=keys.bet_from_private_key
         )
 
+        print(
```
Comment (gabrielfior) [inline examples/replicate_markets/agent_example.py:46]: loguru.debug

File: prediction_market_agent_tooling/markets/omen/omen_replicate.py [hunk 2]
```diff
@@ -74,7 +74,7 @@ def omen_replicate_from_tx(
     existing_categories = set(
         m.category
         for m in get_omen_binary_markets(
-            limit=last_n_omen_markets_to_fetch,
+            limit=1000,
```
Comment (gabrielfior) [inline prediction_market_agent_tooling/markets/omen/omen_replicate.py:77]: Now we are able to fetch all markets (using subgrounds) - is there a reason why we need a hard limit of 1000 here?

Comment (kongzii) [reply prediction_market_agent_tooling/markets/omen/omen_replicate.py:77]: This is just to get what categories are possible on Omen, no need to fetch everything. A sample is okay.

**Review by evangriffiths — APPROVED** [review]
Looks good to me

**Pull request merged**